FreeRTOS/FreeRTOS
# Updated demo_config.h and include paths for OTA demos

Description
-----------


- Fixed the include path for "http_demo_utils.h" for the WinSim ota_over_http_demo.sln. 
- Added the democonfigCLIENT_IDENTIFIER and democonfigMQTT_BROKER_ENDPOINT macros back to the demo_config.h file for both WinSim OTA demos. This is so the E2E test scripts can configure these macros.

Test Steps
-----------

For the failed include...

1. Try to build FreeRTOS\FreeRTOSPlus\Demo\AWS\Ota_Windows_Simulator\Ota_Over_Http_Demo\ota_over_http_demo.sln It will say there is no "http_demo_utils.h" because the include path is absolute.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
